tests/spread/general/stage_snaps/task.yaml

```diff
@@ -1,20 +1,22 @@
 summary: stage-snaps
 
-# We currently only have core18 on a stable channel
 systems:
-  - ubuntu-18.04-64
-  - ubuntu-18.04-amd64
-  - ubuntu-18.04-i386
+  - ubuntu-20.04*
```

Comment:
the architeture specific part of systems can be dropped as the used snaps are no available for all platforms under test. I am weary of this test making our arm64 suite a lot slower than it currently is and have some doubts from the benefits of running these fixtures there, but willing to give it a shot.